docs/release/release-team.md

```diff
@@ -89,10 +90,27 @@ When assembling a release team, the release team lead should look for volunteers
 
 ## Time Commitment
 
-As a member of the release team, you should expect to spend approximately 4-8 hours a week on release related activities for the duration of the term. 
+As a member of the release team, you should expect to spend approximately 4-8 hours a week on release related activities for the duration of the term.
+
+Specific time commitments include:
+   * Release Team meetings once a week throughout the entire release cycle.
+   * Release Day meetings ideally occurring once during the actual release weeks. Refer to release cycle timeline for more specific details.
+   * Any other release-related critical meetings with prior notice.
 
 Before you volunteer to be part of a CAPI release team, please make certain that your employer is aware and supportive of your commitment to the release team.
 
+## Suggestions for Team Leads
+
+  * Clearly communicate with the team shadows you are responsible for, that the majority of the work during the release cycle will be a collaborative effort.
+  * Establish an ownership rotation policy in consultation with respective team members.
+  * Provide opportunities for team members to take the lead in cutting a release within the cycle, based on feasibility.
+  * Define backup ownership and tasks for team members, such as:
+      * Hosting community meetings.
```

Comment:
```suggestion
      * Hosting release team meetings.
```
I assume? Might be good to clarify vs folks could understand this as the CAPI office hours

Reply:
Fixed.